### DISCUSSION TITLE
Todo on the left side of the screen?

### DISCUSSION THREAD

**Discussion by travisgingrich** [rxhanson/Rectangle#380] [Q&A]
Is it possible to leverage todo mode, but with the app on the left side of the screen?

**Comment by rxhanson** [comment]
There's nothing in there right now, but that would be an easy change to add for anyone interested in creating a PR for it.

**Comment by will-stone** [comment]
If this was JS I'd have submitted a PR in an instant, but alas I am no Swift expert. I thought I'd just add to the noise 🗣️ I have been a happy todo-mode user, sitting offset to my screen so that I'm in the middle of the non-todo app space. I have now moved my desk and my laptop screen is now on the other side of my monitor, and so a left-hand todo mode would be very welcome if you get the chance.

<img width="825" alt="image" src="https://user-images.githubusercontent.com/654103/149364871-3a13f3f8-485c-4c18-a481-c00c6d1b7815.png">

**Comment by cromulus** [comment]
Yes, todo mode: AMAZING.
Requests:

- Left side please: right side has desktop icons!
- Choose which monitor: would love to be able to have todo on my secondary monitor, not pinned to primary.

Thanks!

**Comment by sehilyi** [comment]
Would be also great it I can position the to-do app even on the bottom or top of a screen. I use a portrait-mode display and would prefer to place the to-do app on the bottom.

![Screen Shot 2022-07-27 at 13 13 24](https://user-images.githubusercontent.com/9922882/181308561-3a2ae9c0-a7f1-467f-b3fe-af37f39a7f42.png)

(I really like the new to-do feature!)
